tests/server/core/error-pages.js

```diff
@@ -129,12 +129,37 @@ describe('Templating error pages', function () {
 			retval = ret;
 		});
 
-		renderer.render.firstCall.yield(new Error('bad times'));
+		renderer.render.firstCall.yield(new Error('renderer.render threw some error'));
 
 		assert.strictEqual(renderer.render.callCount, 1);
 		assert.strictEqual(null, retval);
 	});
 
+	it('Should callback with null if error falsy', function () {
+		var errorPages = require(moduleName)(config);
+		var retval = false;
+		var error;
+		errorPages.getPage(error, req, res, function (ret) {
+			retval = ret;
+		});
+
+		assert.strictEqual(null, retval);
```

Comment:
See earlier comment on method signature ordering, it should be `actual, expected`